docs/program/sdks/use-extra-params.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Using Extra Params with Viam's SDKs"
+linkTitle: "Extra Params"
+weight: 40
+type: "docs"
+description: "Using extra parameters with Viam's SDKs."
```

Comment:
```suggestion
description: "Use the extra parameter on the component API methods of modular resources."
```

Though I think it's not only component api methods that support it?

Reply:
Made more general with resource